pkg/cpu/cpu.go

```diff
@@ -37,15 +66,20 @@ const (
 
 // Load holds CPU utilization % measured in three intervals of 200ms each
 type Load struct {
-	Avg   string `json:"avg"`
-	Max   string `json:"max"`
-	Min   string `json:"min"`
-	Error string `json:"error,omitempty"`
+	Avg   float64 `json:"avg"`
+	Max   float64 `json:"max"`
+	Min   float64 `json:"min"`
+	Error string  `json:"error,omitempty"`
 }
 
 type counter struct{}
 
-// GetLoad returns the CPU utilization % of the current process
+// GetHistoricLoad returns the historic CPU utilization of the current process
+func GetHistoricLoad() Load {
+	return *rollingAvg
+}
+
+// GetLoad returns the CPU utilization of the current process
```

Comment:
It is not very clear on how this method calculates the CPU load at least to me, can you add some comments in the code below on how it works, e.g. how `end.Sub(start)` corresponds to CPU load

Reply:
Ok, i'll do that.

Reply:
Done